scripts/package.sh

```diff
@@ -82,6 +82,26 @@ if [[ "${HOST_TRIPLE}" != "x86_64-pc-windows-msvc" ]] ; then
     #cp -R rust/build/${HOST_TRIPLE}/llvm/lib/python* deploy/llvm/lib/
 fi
 
+# Sign macOS binaries
+if [[ $HOST_TRIPLE == *apple-darwin* ]]; then
```

Comment:
this has to be outside of `package.sh` and has to be optional. 

In the current state, I will not be able to build this locally since signing will fail.

Reply:
I think you already use some local variables as part of singing process to communicate the key. You can just check whether they are set and not sign if they are not.